R/pkg/R/stats.R

```diff
@@ -33,7 +32,7 @@ setOldClass("jobj")
 #'         of `col2`. The name of the first column will be `$col1_$col2`. Pairs that have no
 #'         occurrences will have zero as their counts.
 #'
-#' @rdname statfunctions
+#' @rdname crosstab
```

Comment:
can we have a `@family stat_functions` here to point users to other stat functions ?